### PR TITLE
Remove power_state argument as agurment (Only as computed property)

### PR DESCRIPTION
**Pull request by marinsalinas**

### CODE REVIEW

File: nutanix/data_source_nutanix_virtual_machines_test.go [hunk 1]
```diff
@@ -27,17 +27,13 @@ func TestAccNutanixVMSDataSource_basic(t *testing.T) {
...
-#   metadata = {
-#     length = 2
-#   }
-# }
+#data "nutanix_clusters" "clusters" {}
 
-# output "cluster" {
-#   value = "${data.nutanix_clusters.clusters.entities.0.metadata.uuid}"
-# }
+#output "cluster" {
```
Comment (thetonymaster) [inline nutanix/data_source_nutanix_virtual_machines_test.go]: Why do we have this commented?

File: nutanix/data_source_nutanix_virtual_machines_test.go [hunk 2]
```diff
@@ -48,11 +44,10 @@ func testAccVMSSDataSourceConfig(rNumVM1 int, rNumVM2 int) string {
 #   num_vcpus_per_socket = 1
 #   num_sockets          = 1
```
Comment (thetonymaster) [inline nutanix/data_source_nutanix_virtual_machines_test.go]: Why are these arguments commented?

File: nutanix/data_source_nutanix_virtual_machines_test.go [hunk 3]
```diff
@@ -48,11 +44,10 @@ func testAccVMSSDataSourceConfig(rNumVM1 int, rNumVM2 int) string {
...
 #   memory_size_mib      = 186
-#   power_state          = "ON"
 
-# }
+#}
 
-# resource "nutanix_virtual_machine" "vm2" {
+#resource "nutanix_virtual_machine" "vm2" {
```
Comment (thetonymaster) [inline nutanix/data_source_nutanix_virtual_machines_test.go]: Why is this VM resource commented out?

File: nutanix/resource_nutanix_virtual_machine.go [hunk 4]
```diff
@@ -1296,6 +1290,8 @@ func resourceNutanixVirtualMachineExists(d *schema.ResourceData, meta interface{
 }
 
 func getVMResources(d *schema.ResourceData, vm *v3.VMResources) error {
+	vm.PowerState = utils.StringPtr("ON")
```
Comment (thetonymaster) [inline nutanix/resource_nutanix_virtual_machine.go]: Should be better to use `ON` as a constant from the client package?

Comment (marinsalinas) [reply nutanix/resource_nutanix_virtual_machine.go]: Good point but this could be a future improvement.

Comment (JonKohler) [reply nutanix/resource_nutanix_virtual_machine.go]: If we're going to do that, lets make sure we create an issue to track as a future enhancement so it doesn't get lost in pull request comments please.

File: nutanix/data_source_nutanix_virtual_machines_test.go (hunk 3 above)
Comment (marinsalinas) [reply nutanix/data_source_nutanix_virtual_machines_test.go]: It's commented because we are going to use it when filters are working.

**Pull request merged**